src/go/rpk/pkg/tuners/executors/commands/create_sized_file_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// +build linux
+
+package commands_test
+
+import (
+	"bufio"
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/tuners/executors/commands"
+)
+
+func TestWriteSizedFileCmdRender(t *testing.T) {
```

Comment:
I'm not really seeing what this is testing / how this test is useful?

Reply:
Most of the tuners (all of them, if I recall correctly) don't do things directly but rather use [commands](https://github.com/vectorizedio/redpanda/tree/dev/src/go/rpk/pkg/tuners/executors/commands), which can be run by 2 different [executors](https://github.com/vectorizedio/redpanda/tree/dev/src/go/rpk/pkg/tuners/executors):
- A direct one, which actually runs the command (makes a syscall, execs a shell command, etc).
- A script-rendering one, which prints the commands on a script so the user can audit and run them anywhere without rpk.

This test checks the latter, for this specific command.

Reply:
The default executor is the direct one, but the script one is used if `--output-script` is passed to `rpk redpanda tune`.